src/dashboard/cd-dashboard-children.vue

```diff
@@ -1,16 +1,56 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-children">
-      <h1 class="cd-dashboard-children__header">My Kids</h1>
+      <h1 class="cd-dashboard-children__header">{{ $t('My Children') }}</h1>
+      <div class="cd-dashboard-children__child" v-for="child in children">
+        <h3>{{ child.firstName }} {{ child.lastName }}</h3>
```

Comment:
Oh, actually... I think we have `child.name` available that has them merged already.